packages/assets-controllers/src/index.ts

```diff
@@ -66,9 +66,8 @@ export type {
 export { TokenListController } from './TokenListController';
 export type {
   Token,
-  TokenRatesConfig,
   ContractExchangeRates,
```

Comment:
all exports from the controller should be included (except AllowedActions and AllowedEvents)